### ISSUE TITLE
At the start of term - configure a start date in the coursebook repo, calculate date range for each week on weekly schedule page

### ISSUE THREAD

**Issue by reubengt** [foundersandcoders/coursebook#606]
The cohort doesn't look into the future that much except during the final projects, when product handover becomes a looming deadline.

I send the coursebook to mentors and speakers quite often and it would be helpful to see the dates for a certain week at a glance 

I'm thinking an env variable that can be updated at the start of term and then just adds 7 days

**Comment by oliverjam** [comment]
Yeah I've wanted to do this for ages, thanks for documenting it 😅 

Related thing that would be nice: a short link like `/today` that redirects to the correct timetable based on the current date
